dragonchain/transaction_processor/level_5_actions.py

```diff
@@ -107,6 +109,25 @@ def execute() -> None:
     # Only check confirmations if there are any pending
     check_confirmations()
 
+    # go through the failed claims backlog
+    _log.info("Trying to finalize any previously failed claims...")
+    process_claims_backlog()
+
+
+def process_claims_backlog() -> None:
```

Comment:
this function definitely needs tests. Just looking at it, I can tell it won't run

Reply:
Adding tests.